sjb/config/test_cases/test_branch_origin_extended_conformance_install_update_containerized.yml

```diff
@@ -0,0 +1,160 @@
+    - type: "script"
+      title: "install origin"
+      repository: "aos-cd-jobs"
+      script: |-
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e containerized=true      \
+                         -e deployment_type=origin  \
+                         /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e containerized=true      \
+                         -e deployment_type=origin  \
+                         -e openshift_image_tag="$( cat ./ORIGIN_TAG )"                      \
+                         -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"            \
+                         -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_TAG )" \
+                         -e openshift_node_port_range='30000-32000'                          \
+                         -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}'   \
+                         /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
+    - type: "script"
+      title: "ensure built version of origin is installed"
+      timeout: 600
+      repository: "aos-cd-jobs"
+      script: |-
+        docker_image_tag="$( cat ./ORIGIN_TAG )"
+        docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
+        docker inspect "${docker_repository}:${docker_image_tag}"
+    - type: "script"
+      title: "update origin to release"
+      repository: "aos-cd-jobs"
+      timeout: 1800
+      script: |-
+        ansible-playbook -vv                    \
+                         --become               \
+                         --become-user root     \
+                         --connection local     \
+                         --inventory sjb/inventory/ \
+                         -e containerized=true      \
+                         -e deployment_type=origin                  \
+                         -e openshift_image_tag="$( cat ./ORIGIN_TAG )"                      \
+                         -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )" \
+                         -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_COMMIT )" \
+                         -e openshift_node_port_range='30000-32000'                    \
+                         -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}' \
+                         -e openshift_disable_check="disk_availability,memory_availability,package_availability,docker_image_availability,docker_storage,package_version" \
```

Comment:
only health checks in place currently are

      - disk_availability      
      - memory_availability    
      - docker_image_availability     

Of these, seems like it would still be a good idea to run the last one.

If the intent is to disable them all so as not to deal with them ever, there's:

`-e openshift_disable_check="*"`

Reply:
I would like to run as many as we can. Though, it takes same time to reach the checks and determine which passed and which failed. Actually, it is the last one that failed.